src/webots/nodes/utils/WbObjectDetection.hpp

```diff
@@ -46,6 +46,9 @@ class WbObjectDetection {
   bool recomputeRayDirection(WbSolid *device, const WbVector3 &devicePosition, const WbMatrix3 &deviceRotation,
                              const WbMatrix3 &deviceInverseRotation, const WbAffinePlane *frustumPlanes);
 
+  // Checks whether the object is detected.
+  // \n\n
+  // TODO: Check whether the parameters are needed as they are already provided in the class constructor.
```

Comment:
Can we address this TODO in this PR?

Reply:
Yes, opening an issue would be nice otherwise I fear it will never be addressed.